charts/hedera-json-rpc-relay/templates/configmap.yaml

```diff
@@ -7,5 +7,10 @@ metadata:
     {{ include "json-rpc-relay.labels" . | nindent 4 }}
 data:
   {{- range $key, $value := .Values.config }}
-    {{ $key }}: {{ if typeIs "float64" $value }}{{ $value | int64 | quote }}{{ else }}{{ $value | quote }}{{ end }}
+    {{- if eq $key "HEDERA_NETWORK" }}
+      {{ $key }}: {{ if typeIs "string" $value}} {{ $value | lower | quote }} {{else}} {{ printf "{%s}" (tpl (join "," $value | trimPrefix "[" | trimSuffix "]") $) | squote }} {{end}}
```

Comment:
Q: should we move this to the template file to allow for commenting also?

Reply:
this is already a template and already allows comments 😅 
Yaml allows comments with #
should I add some comments for clarity?